src/mnemonote.web.server/Models/ValueObjects/ValueObject.cs

```diff
@@ -16,7 +16,7 @@ public override bool Equals(object? obj)
 
             var other = (ValueObject)obj;
 
-            return this.GetEqualityComponents().SequenceEqual(other.GetEqualityComponents());
+            return GetEqualityComponents().SequenceEqual(other.GetEqualityComponents());
```

Comment:
I am missing unit test for this method

Also (matter of taste)
```suggestion
            return GetEqualityComponents()
                .SequenceEqual(other.GetEqualityComponents());
```

or even

```suggestion
            var otherEqualityComponents = other.GetEqualityComponents();
            return GetEqualityComponents()
                .SequenceEqual(otherEqualityComponents);
```

For performance wise it does not matter to introduce new variable (it will be optimized by compiler) but for readability and debugging experience it's better (for me). This is very heavily opinionated. This should be tweaked to your liking

Reply:
unit tests will go to different pr and thank you for suggestion